web/src/components/Exchange/index.js

```diff
@@ -266,7 +269,7 @@ const ExchangePage = ({ points, setPage }) => {
             .replace("{value}", targetCoupon.name.split('_')[1])
             .replace("{points}", (points - targetCoupon.points)))}
           <Cancel onClick={handleBack}>{langText("CANCEL")}</Cancel>
-          <Button onClick={handleExchange}>{langText("CONFIRM")}</Button>
+          <Button onClick={handleExchange((points - targetCoupon.points))}>{langText("CONFIRM")}</Button>
```

Comment:
this should be `{() => handleExchange(points - targetCoupon.points)}` otherwise it will run while render